docs/docs/v0-upgrade/index.md

```diff
@@ -63,9 +63,22 @@ point and we recommended you start there.
 | **Mix Producer**  | [Scenario 2](#scenario-2)            | [Scenario 5](#scenario-5) | [Scenario 8](#scenario-8)            |
 | **v1.0 Producer** | [Scenario 3](#scenario-3)            | [Scenario 6](#scenario-6) | [Scenario 9](#scenario-9) (All v1.0) |
 
-<!-- source https://docs.google.com/drawings/d/137EObOVhMIVk9NEWOX0u_1eQOqe0MRkPQlmwKXkYWEU/edit -->
-
-![Upgrade flows to OPA v1.0](opa-v0-upgrade.png)
+```mermaid
+graph LR
+    start["Recommended<br/>Start Point"] --> s1
+    style start fill:none,stroke:none
+    s10 --> s1["Scenario <br> OPA v0.x"]
+    s1 --> s2
+    s2 --> s3
+    s3 --> s6
+    s6 --> s9
+    s4 --> s1
+    s4 --> s6
+    s5 --> s9
+    s7 --> s9
```

Comment:
In the original picture `Scenario` was spelled out for each one, maybe good to keep it like that?

Reply:
Ahh yeah, I forgot that! https://github.com/open-policy-agent/opa/pull/7618